apps/ledger-live-mobile/android/app/src/main/java/com/ledger/live/newarchitecture/modules/MainApplicationTurboModuleManagerDelegate.java

```diff
@@ -0,0 +1,40 @@
+package com.ledger.live.newarchitecture.modules;
```

Comment:
This file isn't used yet as we are not activating the new RN architecture (Fabric + TurboModules)